contrib/endpoints/src/api_manager/config.h

```diff
@@ -40,9 +40,19 @@ class Config {
   // any of protobuf format json, binary or text. It can be nullptr if there is
   // not server_config.
   static std::unique_ptr<Config> Create(ApiManagerEnvInterface *env,
+                                        const std::string &service_config);
+  // For unit test only
```

Comment:
Just the object constructions.   test codes are using a lot of mocking.  this is to make of object creating easier.  it should not impact the main feature testing.